server/src/test/java/org/elasticsearch/action/admin/cluster/node/stats/NodeStatsTests.java

```diff
@@ -555,11 +555,31 @@ public static NodeStats createNodeStats() {
             List<ScriptContextStats> stats = new ArrayList<>(numContents);
             HashSet<String> contexts = new HashSet<>();
             for (int i = 0; i < numContents; i++) {
+                long compile = randomLongBetween(0, 1024);
+                ScriptContextStats.TimeSeries compileSeries = null;
```

Comment:
This shouldn't be null, let's instead construct an object with all zeros in the false case below.

Reply:
Done.